### DISCUSSION TITLE
Cannot see terminal text in Safari

### DISCUSSION THREAD

**Discussion by saagarjha** [community/community#4894] [Codespaces] [answered]
I'm not entirely sure if this is a Codespaces-specific issue or an xterm-js one, but I can't see anything in the terminal when I use Codespaces in Safari:
<img width="976" alt="Screen Shot 2021-08-02 at 17 53 42" src="https://user-images.githubusercontent.com/13786931/127941099-9325e34d-0143-4994-9d16-3f33c80bc689.png">
The cursor moves around as it normally should and I can copy the output, but the text just isn't visible. I'm seeing this in a fresh codespace with the default settings, so I'm hoping it's not a faulty preference somewhere, but if you suggest a couple to mess with I can see if that improves things.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @saagarjha 
Can you try by updating the following setting to `"terminal.integrated.gpuAcceleration": "off" `?
Here is a link to why that might help: https://code.visualstudio.com/docs/editor/integrated-terminal#_why-is-my-terminal-showing-a-multicolored-triangle-or-a-completely-black-rectangle

Please let us know if that helps resolve your issue.

**Reply by saagarjha** [reply]
Yep, it does! I can work with this, but if there is anything I can do to help with why GPU acceleration is not working do let me know.

**Reply by dzsquared** [reply]
Thank you, @tanmayeekamath  - this was able to fix the same issue for me.  I’m on Safari in padOS 15 and the lack of visible text in the terminal was tough to work around.
